src/Core/Exception/Exception.php

```diff
@@ -9,107 +9,8 @@
  * Redistributions of files must retain the above copyright notice.
  *
  * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
- * @since         3.0.0
+ * @since         4.2.0
```

Comment:
Shouldn't this remain 3.0.0 and instead the new class have 4.2.0?

Reply:
Up to you. I thought maybe we would say the alias was since 4.2 but the actual exception code since the original version?

I'll switch if you're sure.